test/rpc_test.py

```diff
@@ -90,16 +89,21 @@ def _wrap_with_rpc(test_method):
 
     @functools.wraps(test_method)
     def wrapper(self, *arg, **kwargs):
-        store = dist.FileStore(self.file_name, self.world_size)
-        dist.init_process_group(
-            backend="gloo", rank=self.rank, world_size=self.world_size, store=store
+        init_method = (
+            "file://{file_name}?rank={rank}&world_size={world_size}"
```

Comment:
nit, if we plan to use this string a lot, should we save this string format in a global to avoid having to rewrite it? Such as:
```
INIT_STR= "file://{file_name}?rank={rank}&world_size={world_size}"
```

then to use:
`dist.init_process_group(backend=Backend.GLOO, init_method=INIT_STR.format(...))`

Reply:
@rohan-varma 
Let me do it.